rest/admin_api.go

```diff
@@ -478,7 +479,8 @@ func (h *handler) deleteRole() error {
 
 func (h *handler) getUserInfo() error {
 	h.assertAdminOnly()
-	user, err := h.db.Authenticator().GetUser(internalUserName(mux.Vars(h.rq)["name"]))
+	eName := url.QueryEscape(mux.Vars(h.rq)["name"])
```

Comment:
Wouldn't this break for existing usernames with escapable characters that are currently valid in a username?  (e.g. `+`, `@`, `%`)?